oshi-core/src/main/java/oshi/hardware/PowerSource.java

```diff
@@ -48,4 +49,31 @@
      *         or -2.0 (unlimited)
      */
     double getTimeRemaining();
+    
+    /**
+     * Power source health as a fraction of the battery's designed capacity.
+     * 
+     * @return A value between 0.0 (no capacity) and 1.0 (at designed capacity)
+     */
+    double getHealth();

```

Comment:
Is "Health" a reasonably consistent term?  I googled and found [State of Health (SOH)](https://en.wikipedia.org/wiki/State_of_health) which looks like exactly what you're trying to represent here.  I think `getHealth()` is probably ok but if SOH is really a more common/standard term I might prefer a `getStateOfHealth()` method.